src/polyfills/index.js

```diff
@@ -14,25 +14,25 @@
  * limitations under the License.
  */
 
-/** @fileoverview */
+/** @fileoverview Installs polyfills depending on build environment. */
```

Comment:
Looks like there are [~300 comments](https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=%22src%2Fpolyfills.js%22) that read `// src/polyfills.js must be the first import.`, that could do with an update. They mostly live in 3p code.

Reply:
Yep I've seen em all. I plan to update all of those in a separate PR so Justin or Kris can rubber-stamp without needing to look